internal/hierarchyconfig/reconciler_test.go

```diff
@@ -32,7 +32,7 @@ var _ = Describe("Hierarchy", func() {
 		fooName = CreateNS(ctx, "foo")
 		barName = CreateNS(ctx, "bar")
 		config.SetNamespaces("")
-		config.SetManagedMeta(nil, nil)
+		Ω(config.SetManagedMeta(nil, nil)).Should(Succeed())
```

Comment:
As cute as it is, we don't use `Ω` anywhere else in our codebase. Can we just use `Expect()` here instead?